authorization/token/service/token_service_blackbox_test.go

```diff
@@ -491,3 +491,306 @@ func (s *tokenServiceBlackboxTest) TestAuditNonExistentResource() {
 	require.Error(s.T(), err)
 	require.IsType(s.T(), err, errors.BadParameterError{})
 }
+
+func (s *tokenServiceBlackboxTest) TestTokenUpdatedWhenUserAcceptsResourceInvitation() {
```

Comment:
could you split the test if you cover multiple cases?

Reply:
This is a complex test that continuously builds upon and depends on state initialized by the previous steps.  It's not really feasible to split it up.

Reply:
ok, I did not realize that the previous steps mattered.